lexicons/com/atproto/resource/authentication.json

```diff
@@ -0,0 +1,24 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.resource.authentication",
```

Comment:
went with `authentication` here instead of `serviceAuth` since i felt it was cleaner & matched the other schemas better stylistically